cmd/exo/cmd/ssh.go

```diff
@@ -48,12 +64,27 @@ type sshInfos struct {
 	vmID     string
 }
 
-func getSSHInfos(name string) (*sshInfos, error) {
+func getSSHInfos(name string, isIpv6 bool) (*sshInfos, error) {
 	vm, err := getVMWithNameOrID(cs, name)
 	if err != nil {
 		return nil, err
 	}
 
+	nic := vm.DefaultNic()
+	if nic == nil {
+		return nil, fmt.Errorf("No default NIC on this instance")
+	}
+
+	vmIP := vm.IP()
+
+	if isIpv6 {
+		if nic.IP6Address.String() != "<nil>" {
```

Comment:
`nic.IP6Address != nil`

Reply:
in this case `nic.IP6Address` is not `nil`

Reply:
then the opposite...